signup.css

```diff
@@ -0,0 +1,114 @@
+*{
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+body{
+  background-color: #f7f4f4;
+  font-family:sans-serif;
+}
+.blue{
+  color:#1F59BB;
+  text-transform: capitalize;
+  font-weight: 400;
+}
+.info{
+  font-size: 13px;
+  font-weight: 600;
+  text-transform: capitalize;
+}
+.signUp{
+  padding:0 .2rem;
+}
+.privacy{
+  font-weight: 300;
+  text-align: center;
+  margin-top: 1rem;
+  font-size: 12px;
+}
+.submit{
+  text-transform: capitalize;
+  text-align: center;
+  color:#fff;
+  width: 100%;
+  background-color: #1F59BB;
+  border:transparent;
+  margin-top: .8rem;
+  padding:.6rem 0;
+  border-radius: 10px;
+}
+.submit::after{
+  content: '!';
+}
+
+.signIn{
+  text-align: center;
+  margin-top: 1.2rem;
+  display: flex;
+  align-self: center;
+  justify-content: center;
+}
+
+.signIn span{
+  padding-left: .3rem;
+}
+
+span a{
+  color:darkblue;
+  font-weight: 500;
+}
+
+input{
+  border: 1px solid #1F59BB;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#1F59BB" to be "#1f59bb" (color-hex-case)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5576441)